@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -263,9 +298,20 @@ export class AltitudeBasedClipPlanesEvaluator implements ClipPlanesEvaluator {
         // The distance to tangent point may be descibed as:
         // t = sqrt(d^2 - r^2)
         const t = Math.sqrt(d * d - r * r);
+        // Because we would like to see elevated geometry that may be visible beyond the tangent
+        // point on ground surface, we need to extend viewing distance along the tangent line
+        // by te (see graph above). Knowing that OTE forms right angle, we have:
+        // (r+e)^2 = r^2 + te^2
+        // te = sqrt((r+e)^2 - r^2)
+        // This reduces to:
+        // te = sqrt(r^2 + 2*r*e + e^2 - r^2)
+        // te = sqrt(2*r*e + e^2)
+        const te = Math.sqrt(2 * r * this.maxElevation + this.maxElevation * this.maxElevation);
```

Comment:
The effect of 'te' - tangent extension is visible in the screen-shot attached to the ticket (HARP-6270). They show how much far plane is moved beyond the "normal" sphere tangent point, and it will allow to see high mountains on the sphere curvature (quite nice feature), otherwise such mountains (or other elevated features) would just pop and slice on the horizon.

Reply:
I added a comment on the ticket.

Reply:
Responded to it.